spec/concurrent/edge/future_spec.rb

```diff
@@ -171,6 +171,12 @@
       expect([queue.pop, queue.pop, queue.pop, queue.pop].sort).to eq [:async, :async, :sync, :sync]
     end
 
+    it 'supports setting timeout while waiting' do
+      future = Concurrent.future { sleep 1 }
```

Comment:
The sleep w/i the block shouldn't effect the speed of the tests. This test never waits for the future to finish. The test will end after the 0.1 second wait on line 176.